scripts/src/javatests/org/oppia/android/scripts/ci/ComputeChangedFilesTest.kt

```diff
@@ -296,28 +289,33 @@ class ComputeChangedFilesTest {
   }
 
   @Test
-  fun testUtility_developBranch_instrumentationModuleChanged_instrumentationFilesAreIgnored() {
+  fun testUtility_developBranch_instrumentationModuleChanged_filteredFilesAreIgnored() {
     initializeEmptyGitRepository()
-    createFiles(
+    createAndCommitFiles(
       "InstrumentationFile",
       subPackage = "instrumentation/src/javatests/org/oppia/android/instrumentation/player"
     )
-    createFiles(
+
+    val reportedFiles = runScript()
+
+    assertThat(reportedFiles).hasSize(0)
```

Comment:
```suggestion
    assertThat(reportedFiles).isEmpty()
```

Ditto elsewhere in the test file.

Reply:
Done, updated as suggested.